app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Controller with no actions?

Reply:
The empty controllers were generated by devise by default. I thought of overriding them so i included them in the app.